### ISSUE TITLE
Input fields lack visible labels

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#712]
**Description of issue**

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 3.3.2 Labels or Instructions (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions).

Input fields should have persistently visible labels that describe their purpose - a placeholder text that disappears as soon as users start typing isn't a sufficient substitute.

The message field and emoji search field both rely on nothing but placeholder text for this purpose.

![Message input field lacking a visible label, screenshot.](https://github.com/user-attachments/assets/887b38b3-974b-4d39-ba94-c1caf6b9bdd8)
![Emoji search field lacking a visible label, screenshot.](https://github.com/user-attachments/assets/f45caca5-25d4-4123-975a-213fd1173f79)

**To Reproduce**

1. Find the message input field and the emoji search field

**Recommended solution**

Add a visible field label. In the code example below, it's also clickable and correctly linked to its field:

Code example:
```
<label for="search">Search</label>
<input id="search" type="text">
```

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: TBA

**Desktop:**

 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).
